pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -2022,6 +2025,31 @@ SELECT objoid, classoid, provider, label FROM pg_catalog.pg_shseclabel
 ----
 objoid  classoid  provider  label
 
+subtest pg_catalog.pg_aggregate
+
+query O colnames
+SELECT oid::REGPROC FROM pg_proc WHERE proisagg = true EXCEPT SELECT aggfnoid FROM pg_aggregate
+----
+oid
+
+# Check whether correct operator's oid is set for max and bool_or.
+query OTO colnames
+SELECT c.oid, c.oprname, a.aggsortop FROM pg_aggregate a
+JOIN pg_proc b ON a.aggfnoid = b.oid::REGPROC
+JOIN pg_operator c ON c.oprname = '>' AND b.proargtypes[0] = c.oprleft AND b.proargtypes[0] = c.oprright
+WHERE (b.proname = 'max' OR b.proname = 'bool_or') AND c.oid != a.aggsortop;
+----
+oid  oprname  aggsortop
+
+# Check whether correct operator's oid is set for min, bool_and and every.
+query OTO colnames
```

Comment:
These tests look good but I don't quite understand why they don't return any values. A better test would probably produce the oid for min, bool_and and every. same goes for the above.

Reply:
Sure